util/reggen/README.md

```diff
@@ -59,9 +59,45 @@ $ cd $REPO_TOP/util
 $ mkdir /tmp/dv
 $ ./regtool.py -s -t /tmp/dv ../hw/ip/uart/data/uart.hjson
 $ ls /tmp/dv
-    uart_reg_block.sv
+    uart_ral_pkg.sv
 ```
 
+By default, the generated block, register and field models are derived from
```

Comment:
@sha-ron: please see this description of how this would work.

Reply:
thanks, tried it and looks good